code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -333,6 +345,11 @@
 	imp.implant(src, src) //acts as if the abno is both the implanter and the one being implanted, which is technically true I guess?
 	datum_reference.abno_radio = TRUE
 
+/mob/living/simple_animal/hostile/abnormality/proc/QliphothDeterrence() //Are you in a cell and currently contained?? If so stop.
```

Comment:
A very vague proc name. We could change it to "InContainment()" and use it instead of checking for status flags everywhere.